dask_sql/physical/rex/core/call.py

```diff
@@ -600,6 +600,21 @@ def extract(self, what, df: SeriesOrScalar):
             raise NotImplementedError(f"Extraction of {what} is not (yet) implemented.")
 
 
+class ToTimestampOperation(Operation):
+    def __init__(self):
+        super().__init__(self.to_timestamp)
+
+    def to_timestamp(self, df, format):
+        # Remove double and single quotes from string
+        format = format.replace('"', "")
+        format = format.replace("'", "")
+
+        df = pd.to_datetime(df, unit="s")
```

Comment:
Depending on the outcome of the above modifications to `dsql_totimestamp`'s signature, we may not need this casting to datetimes